web/src/components/tasks/business-formation/business/BusinessStep.test.tsx

```diff
@@ -812,44 +812,62 @@ describe("Formation - BusinessStep", () => {
       const legalStructureId = "limited-liability-company";
       await getPageHelper({ legalStructureId }, {});
```

Comment:
[sand] These tests are really repetitive and could probably use an it.each to iterate through instead

Reply:
[sand] also a bit should we also test for foreign in the same fashion here?